Makefile

```diff
@@ -39,9 +39,9 @@ install-dependencies:
 	pip install --upgrade $(DEVPKGS)
 
 ## sharedobj   : build khmer shared object file
-sharedobj: khmer/_khmermodule.so
+sharedobj: khmer/_khmer.so
```

Comment:
This is not true for py3, since it creates the dynamic library as `_khmer.cpython-34m.so`, for example.